autogpt_platform/frontend/src/components/auth/AuthFeedback.tsx

```diff
@@ -1,15 +1,117 @@
+import { AlertCircle, CheckCircle } from "lucide-react";
+import { Card, CardContent } from "@/components/ui/card";
+import { HelpItem } from "@/components/auth/help-item";
+import Link from "next/link";
+import { BehaveAs } from "@/lib/utils";
+
 interface Props {
   message?: string | null;
   isError?: boolean;
+  showErrorPrompt?: boolean;
```

Comment:
is there a case where we set `showErrorPrompt` = true when `isError` = False or vice-versa ?

Reply:
maybe? they were seperate before i belive